Tests/Shared/TestRunnerEnvironment.swift

```diff
@@ -4,16 +4,18 @@
 
 import Foundation
 
-struct TestRunnerEnvironment {
+enum TestRunnerEnvironment {
     /// `true` if the tests are currently running on the CI. We get this information by checking for the custom `CI` environment
     /// variable passed in to the process.
     static var isCI: Bool {
         ProcessInfo.processInfo.environment["CI"] == "TRUE"
     }
     
-    /// `true` if the current tests are stress test. We get this information by checking for the custom `STRESS_TESTS` environment
-    /// variable passed in to the process.
-    static var isStressTest: Bool {
-        ProcessInfo.processInfo.environment["STRESS_TESTS"] == "TRUE"
+    /// Number of stress test invocations
+    static var testInvocations: Int {
+        ProcessInfo.processInfo.environment["TEST_INVOCATIONS"].flatMap(Int.init) ?? 1
```

Comment:
Just curious... why 1 test invocation when there are no tests??

Reply:
Not sure I follow, what do you mean by "there are no tests"? 😄 1 is default value if environment variables do not contain `TEST_INVOCATIONS` (e.g. default test plan) or the value is not `Int` convertible